android/app/build.gradle.kts

```diff
@@ -113,6 +113,7 @@ emerge {
 
 sentry {
   org.set("emerge-tools")
```

Comment:
Does the org need to be updated too? Actually what happens if you specify an org different from the auth token org?